src/include/libpbs.h

```diff
@@ -405,7 +407,7 @@ void *get_conn_svr_instances(int);
 int pbs_register_sched(const char *sched_id, int primary_conn_id, int secondary_conn_id);
 int get_svr_inst_fd(int vfd, char *svr_inst_id);
 int random_srv_conn(svr_conn_t **);
-int starting_index(char *);
+int get_svr_idx_from_jid(char *);
```

Comment:
I would rather call this function as get_job_location_hint() or something similar. In other words, I suggest we not give away the implementation in the function name. All we are saying is that this will take a void * and return some kind of a server location hint - which may or may not be correct - and we might have to loop over servers.

Reply:
I've renamed the function, but I'd like to avoid void pointers, I can't really foresee this function ever accepting a type other than char *. We can change the signature if/when needed. What do you think?

Reply:
how about "get_expected_svr_idx" or "get_original_svr_idx" or "get_probable_svr_idx"? get_..._hint() makes it sound like the function is returning a hint, where it is actually returning the potential server index

Reply:
Well i was trying to intentionally keep the function name vague/abstract, so that future implementations could use it more like an hint - 

potential server index = hint --> i think both are the same things (hint being hint about the server index) but if you like the word probable more, i am fine with it...

Reply:
ok, i understand your POV now, I'll make it "get_job_location_hint"

Reply:
Just for a thought way down the road, void *'s are a little bit of a pain in C++.  You have to static_cast them at the use.
(e.g. static_cast<char *>(malloc())).  We're not converting the server to C++ any time soon, but it is something to think of.

Reply:
im just going to rename, not convert the argument to void *, we can do it if/when needed to support multiple different naming schemes, I'd like to wait until we design that.